app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -380,6 +381,21 @@ fun List<Message>.getFoldersIds(exception: String? = null) = mapNotNull { if (it
 fun List<Message>.getUids(): List<String> = map { it.uid }
 //endregion
 
+fun Fragment.deleteWithConfirmationPopup(folderRole: FolderRole?, count: Int, executeDelete: () -> Unit) {
```

Comment:
Maybe "deleteCallback" instead if "executeDelete" ?  I think we often use callback naming